### PR TITLE
Fix legacy SVGs

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

The legacy SVG loader is no longer needed. This change removes it, and removes the explicit `svg-loader!` pragmas in legacy JavaScript modules. The change also mocks out SVGs in tests.

## What is the value of this and can you measure success?

Less code to maintain. This will also fix an issue whereby SVGs were being double-transformed, once implicitly by Webpack, and once explicitly via the `svg-loader!` in the module.

c/c @Mullefa @dominickendrick 

## Does this affect other platforms - Amp, Apps, etc?

No

## Tested in CODE?

No

### CODE REVIEW

File: static/test/javascripts-legacy/spec/commercial/hosted/about-popup.spec.js [hunk 1]
```diff
@@ -19,7 +19,19 @@ define([
...
             };
 
+
+
         beforeEach(function (done) {
+            var fakeSvg = {
+                markup: ''
+            };
+
+            injector.mock({
```
Comment (regiskuckaertz) [inline static/test/javascripts-legacy/spec/commercial/hosted/about-popup.spec.js]: Would it make sense to override the default `mock` [here](https://github.com/guardian/frontend/blob/master/static/test/javascripts-legacy/helpers/injector.js) for SVGs?

**Review by regiskuckaertz — APPROVED** [review]
Yoohoo!

File: static/test/javascripts-legacy/spec/commercial/hosted/about-popup.spec.js (hunk 1 above)
Comment (SiAdcock) [inline static/test/javascripts-legacy/spec/commercial/hosted/about-popup.spec.js]: And just implicitly mock SVGs whenever you call `injector.mock()`? Maybe. Although the Injector is on the way out as soon as all modules are ES6ified.

**Review by sndrs — APPROVED** [review]

**Comment by dominickendrick** [comment]
:1st_place_medal:

**Pull request merged**